extended-statistics/src/test/java/org/infinispan/stats/topK/LocalTopKeyTest.java

```diff
@@ -79,6 +79,7 @@ public void testGet(Method method) {
       assertWriteSkew(key2, 0);
    }
 
+   @Test(invocationCount = 1000)
```

Comment:
Considering this isn't a stress test, I wouldn't commit this. Maybe with `invocationCount = 100`...